spec.bs

```diff
@@ -1452,7 +1455,12 @@ To <dfn export>generate a [=/pattern string=]</dfn> from a given [=/part list=]
       1. Append the result of running [=generate a segment wildcard regexp=] given |options| to the end of |result|.
       1. Append "`)`" to the end of |result|.
     1. Else if |part|'s [=part/type=] is "<a for=part/type>`full-wildcard`</a>":
-      1. If |custom name| is true:
+      1. If one of the following are true:
+        <ul>
+          <li>|custom name| is true.</li>
```

Comment:
Specs are very inconsistent but my preferred style these days for this sort of thing is

> If one of the following is true:
> 
> - x; or
> - y
> 
> then: